Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM ubuntu:14.04
 
 # environment
 RUN locale-gen "en_US.UTF-8" && DEBIAN_FRONTEND=noninteractive dpkg-reconfigure locales
+ENV DEBEMAIL "sysadmins@cs50.harvard.edu"
+ENV DEBFULLNAME "CS50 Sysadmins"
```

Comment:
Can we avoid having these in here, so that `cs50/cli` can be used by others outside of CS50 without CS50-specific names/addresses therein?